cmd.js

```diff
@@ -101,13 +102,40 @@ try {
   fatal(`Out file isn't writable\n${e.message}`);
 }
 
+// Set up image export directory for formats with image export.
+let saveImage = undefined;
+if (cli.flags.fmt === 'html') {
+  const imgDirname = 'range-export';
+  const imgPath = resolve(join(dirname(cli.flags.out), imgDirname));
+  if (!existsSync(imgPath)) {
+    mkdirSync(imgPath);
+  }
+  saveImage = (name, url) => {
+    const baseName = basename(name);
+    const imgFile = createWriteStream(join(imgPath, baseName));
+    https.get(url, function(resp) {
```

Comment:
What about returning the promise and then serializing to avoid saturating http connections?

Reply:
Sure, I'll give that a shot. I was struggling a bit with harmonizing the different styles of async that some of these libraries use, so I punted on any sort of "wait for this to be done", but I should be able to get something working.

Reply:
Alright, I got it working. The script now waits for image downloads to complete, and shows a progress bar for that phase too. ✨